enigma-tools-m/src/lib.rs

```diff
@@ -1,5 +1,11 @@
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
-#![deny(unused_extern_crates)]
+#![deny(unused_extern_crates, missing_docs)]
+//! # Enigma Tools Mutual
+//! This library provides tools for both untrusted and trusted sides of the SGX. <br>
+//! It should supersede both `enigma-tools-t` and `enigma-tools-u`. <br?
```

Comment:
what did you mean to write here?

Reply:
that we don't need 3 different tools libraries we can merge them all to one (although some of the things there really don't relate to the other side so i'm not so sure)